index.js

```diff
@@ -1,5 +1,48 @@
-import { NativeModules } from 'react-native';
+import { NativeModules } from 'react-native'
+const { Jose } = NativeModules
 
-const { Jose } = NativeModules;
+function pem2der(key) {
+  return key
+    .replace(/(?:\r\n|\r|\n)/g, '')
+    .replace('-----BEGIN RSA PRIVATE KEY-----', '')
+    .replace('-----END RSA PRIVATE KEY-----', '')
+    .replace('-----BEGIN RSA PUBLIC KEY-----', '')
+    .replace('-----END RSA PUBLIC KEY-----', '')
+}
 
-export default Jose;
+function keys(keys) {
+  return { jwk: keys.jwk, der: pem2der(keys.pem) }
+}
+
+export const sign = Jose.sign
```

Comment:
```suggestion
export const sign = async (payload, privateKeys, header) => {
  return Jose.sign(payload, keys(privateKeys), header)
}
```

However this would introduce a breaking change (the app needs to pass the pem to as `pem` property and doesn't need to do `pem2der` )  so the git commit message should be marked with a `BREAKING CHANGE: `